client/src/js/base/InputSave.js

```diff
@@ -97,25 +110,31 @@ export class InputSave extends React.Component {
     };
 
     blur = () => {
+        this.inputNode.blur();
         this.buttonNode.blur();
     };
 
     render () {
+        const formStyle = this.props.noMargin ? "0px" : "15px";
```

Comment:
Change to `formMargin`. The implied convention is that variables with the `Style` suffix are always style objects compatible with React components.

Here you are describing a margin value.

By the way, `0` values for CSS properties can just be 0. No unit is necessary.